### ISSUE TITLE
Downloads not working - Could not find any csrf-token in _prepare_session

### ISSUE THREAD

**Issue by peterk** [iamleot/transferwee#40]
It seems like WeTransfer changed something in the past couple of days and download_url now raises ConnectionError('Could not prepare session'). 

_prepare_session reports "Could not find any csrf-token".

**Comment by peterk** [comment]
The problem seems to be that wetransfer.com no longer has a csrf token to be parsed at line 192 in [_prepare_session](https://github.com/iamleot/transferwee/blob/c5f6984c335ff8b64dea50cb82660aa67da748a9/transferwee.py#LL192C20-L192C37)

**Comment by iamleot** [comment]
Hello! Can you please share a possible way to test it?

At least yesterday and last night all tests passed successfully.

I've also rechecked via `tests/check.sh` now and I could not reproduce it.

**Comment by peterk** [comment]
I get the same error running the check script.

```
(base) ➜  tests git:(master) ./check.sh
Creating a test file...
Uploading the test file...
Checking that all files exists
Checking for no duplicate filenames
Preparing to upload
Could not find any csrf-token
Traceback (most recent call last):
  File "/transferwee_orig/transferwee/transferwee.py", line 566, in <module>
    print(upload(args.files, args.n, args.m, args.f, args.t))
  File "/transferwee_orig/transferwee/transferwee.py", line 485, in upload
    raise ConnectionError('Could not prepare session')
ConnectionError: Could not prepare session
```

Looking at the wetransfer.com website source I see no csrf element to parse.

Buuut...I am connecting from Sweden. Switching to VPN and connecting from the US the csrf token is there. Are they rolling out a new website by region?

**Comment by iamleot** [comment]
I have no idea, sorry! Everything that I know about that was reverse engineered!

That's interesting though! Unfortunately until I will not be able to reproduce I could probably not easily help.

**Comment by peterk** [comment]
Tested various methods and it seems that cookies are used instead of the csrf meta element in some cases. Continuing with the fetched sessions makes the check script work again.

**Comment by iamleot** [comment]
Fixed via #42.

Thanks!
